applications/Chat/evaluate/README.md

```diff
@@ -1,182 +1,260 @@
+The following is an example showing the config structure:
+```
+{
+    "language": "cn",
+    "category": {
+        "brainstorming": {
+            "GPT-3.5": ["relevance", "creativity", "practicality", "correctness"],
+            "Metrics": ["Distinct"]
+        },
+        "chat": {
+            "GPT-3.5": [ "relevance", "naturalness", "engagingness", "reasonableness"],
+            "Metrics": ["Distinct"]
+}
+```
+`"language"`: evaluate the model capability in which language, choosing from `"cn"` (we only support Chinese for now).
+`"category"`: evaluate the model capability in which category/categories, choosing from:
+```
+ "brainstorming"
+ "chat"
+ "classification"
+ "closed_qa"
+ "extraction"
+ "generation"
+ "open_qa"
+ "rewriting"
+ "roleplay"
+ "summarization"
+```
+
+`"GPT-3.5"`: config metrics for GPT-3.5 evaluation, choosing from:
+```
+"language organization"
+"relevance"
+"creativity"
+"practicality"
+"correctness"
+"naturalness"
+"engagingness"
+"reasonableness"
+"diversity"
+"fidelity"
+"conciseness"
+```
+`"Metrics"`: config metrics for automatic metrics evaluation, choosing from:
+```
+"BLEU"
+"ROUGE"
+"Distinct"
+"BERTScore"
+"Precision"
+"Recall"
+"F1 score"
+```
+
+#### Evaluate
+After setting the configuration file, you can evaluate the model using `eval.py`.
+
+
+An example script is provided as follows:
+```shell
+python eval.py \
+    --config_file "path to the config file" \
+    --battle_prompt_file "path to the prompt file for battle" \
+    --gpt_evaluation_prompt_file "path to the prompt file for gpt evaluation" \
+    --target_file "path to the target answer file" \
+    --answer_file_list "path to the answer files of at most 2 models" \
+    --model_name_list "the names of at most 2 models" \
+    --save_path "path to save results" \
+    --openai_key "your openai key" \
+```
+
+## To Do
+- [ ] Add evaluation for English capability
+- [ ] Support UniEval
+- [ ] Support GPT-4 evaluation
+
+## Citations
+
+```bibtex
+@misc{vicuna2023,
+    title = {Vicuna: An Open-Source Chatbot Impressing GPT-4 with 90\%* ChatGPT Quality},
+    url = {https://vicuna.lmsys.org},
+    author = {Chiang, Wei-Lin and Li, Zhuohan and Lin, Zi and Sheng, Ying and Wu, Zhanghao and Zhang, Hao and Zheng, Lianmin and Zhuang, Siyuan and Zhuang, Yonghao and Gonzalez, Joseph E. and Stoica, Ion and Xing, Eric P.},
+    month = {March},
+    year = {2023}
+}
+
+@misc{ouyang2022training,
+      title={Training language models to follow instructions with human feedback},
+      author={Long Ouyang and Jeff Wu and Xu Jiang and Diogo Almeida and Carroll L. Wainwright and Pamela Mishkin and Chong Zhang and Sandhini Agarwal and Katarina Slama and Alex Ray and John Schulman and Jacob Hilton and Fraser Kelton and Luke Miller and Maddie Simens and Amanda Askell and Peter Welinder and Paul Christiano and Jan Leike and Ryan Lowe},
+      year={2022},
+      eprint={2203.02155},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
```

Comment:
Add citation for this [paper](https://arxiv.org/abs/2303.16634).

Reply:
added